share/xtp/xml/decomp.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<options>
+  <decomp help="Allows you to compute localised orbitals using the Pipek-Mezey scheme">
+  <activeatoms help="Indices of atoms in active region">
+    1 5
```

Comment:
remove value and please use the `<a help="" />` syntax if you do not have a value between tags.